.extlib/simplesamlphp/modules/saml/src/Auth/Source/SP.php

```diff
@@ -306,6 +306,22 @@ public function getHostedMetadata(): array
      */
     public function getIdPMetadata(string $entityId): Configuration
     {
+        assert(is_string($entityId));
```

Comment:
This assertion should likely be removed, the typeguard will force it to a string either way. Leftovers from debugging?